mentorship iosTests/ProfileTests.swift

```diff
@@ -148,6 +148,36 @@ class ProfileTests: XCTestCase {
         XCTAssertEqual(profileVM.getProfile().name, "newName")
     }
     
+    func testCanSaveProfileWithoutNameAndFail() throws {
+        // Profile Service
+        let profileService: ProfileService = ProfileAPI(urlSession: urlSession)
+        
+        // View Model
+        let sampleData = ProfileModel.ProfileData(id: 0, name: "", username: "", email: "")
+        let profileVM = ProfileViewModel()
+        profileVM.saveProfile(profile: sampleData)
+        
+        // Profile Editor View
+        let profileEditor = ProfileEditor(profileService: profileService, profileViewModel: profileVM)
+        
+        XCTAssertFalse(profileEditor.canSave)
+    }
+    
+    func testCanSaveProfileWithNameAndSucceed() throws {
+        // Profile Service
+        let profileService: ProfileService = ProfileAPI(urlSession: urlSession)
+        
+        // View Model
+        let sampleData = ProfileModel.ProfileData(id: 0, name: "name", username: "", email: "")
+        let profileVM = ProfileViewModel()
+        profileVM.saveProfile(profile: sampleData)
+        
+        // Profile Editor View
+        let profileEditor = ProfileEditor(profileService: profileService, profileViewModel: profileVM)
+        
+        XCTAssertTrue(profileEditor.canSave)
+    }
+    
```

Comment:
We probably don't need a throwing function. Also, can we improve the naming of these test functions?

Reply:
ok. i changed the name based on another test function that already exists